content/en/agent/faq/agent-downgrade-minor.md

```diff
@@ -1,20 +1,24 @@
 ---
-title: Downgrade the Agent v6 to a prior minor version
+title: Downgrade the Agent to a prior minor version
 kind: faq
 aliases:
 - /agent/faq/downgrade-datadog-agent
 ---
 
-For DEB or RPM packages of the Datadog Agent, find below instructions to downgrade the Datadog Agent to version `6.X.Y`.
+For DEB or RPM packages of the Datadog Agent, find below instructions to downgrade the Datadog Agent to a prior minor version.
+
+To downgrade the Datadog Agent to a prior major version, follow the instructions on the [dedicated page][1].
 
 **Note**: These instructions only work for Datadog Agent version 6.x and above.
 
+**Note**: These instructions for configuration management tools only work with the latest major version of these tools. If using a prior version, refer to the documentation for this version on the tools' repositories: [Chef cookbook][2], [Puppet module][3], [Ansible role][4].
```

Comment:
I think we should mention here what these major versions are explicitly, both for the current "latest major version" and the previous major versions to which the the links point.

That way:
1. the versions are explicit
2. if we ever forget to update this page after a config mgmt major release it will be easy for us (and users) to realize that the instructions do not work for the actual latest config mgmt tool.

Reply:
I removed `cookbook/role/module` from the tab titles but not from the note, as it makes it clear that we're not talking about the version of Chef / Ansible / Puppet themselves, but of our cookbook/role/module.